tests/unit/type/stream.tcl

```diff
@@ -199,6 +199,15 @@ start_server {
         assert {[r EXISTS otherstream] == 0}
     }
 
+    test {XADD with LIMIT delete entries no more than limit} {
+        r del yourstream
+        for {set j 0} {$j < 3} {incr j} {
+            r XADD yourstream * xitem v
+        }
+        r XADD yourstream MAXLEN ~ 0 limit 1 * xitem v
+        assert {[r XLEN yourstream] >= 3}
```

Comment:
i think it's better to assert the exact return rather than `>=` (also in the other test)

Reply:
The result is based on `stream-node-max-entries`. So no matter what this config is, `>=` is always right.

Reply:
mmm ok i accept, but in the other test you explicitly set `stream-node-max-entries` so maybe you could use `==`

Reply:
I both updated. The first test `stream-node-max-entries` is 100.